### ISSUE TITLE
[@uifabric/example-app-base] Hamburger menu button is not collapsing through Esc key which appears after resizing the browser up to 200% 

### ISSUE THREAD

**Issue by tyveille** [microsoft/fluentui#16552]
### Environment Information

Test Environments:
Version:  2004 (OS Build: 20236.1005)
Browser: Edge chromium 
Tool: Settings

### Describe the issue:

When navigating to a site using example-app-base, low visible user will not able to access the controls of the page efficiently if the hamburger menu item is not collapsing on pressing Esc key which appears after resizing the browser up to 200%

### Please provide a reproduction of the issue in a codepen:

Unable to use codepen as the issue is located in @uifabric/example-app-base

Resize the browser upto 200% and refresh the page.
Navigate on Hamburger menu item and activate it through keyboard.
Press Esc key to collapse the Hamburger menu item
Verify the issue

#### Actual behavior:

When page is resized to 200%, Hamburger menu button is not collapsing through Esc key neither by Enter or Space bar key and on pressing TAB key the focus indicator remains on "Shared Components" button which is appearing under Hamburger menu item

#### Expected behavior:

When page is resized to 200%,, Hamburger menu button should gets collapse through Esc key and on pressing TAB key the focus indicator should move out side of the hamburger menu item

### Documentation describing expected behavior

N/A

**Comment by gouttierre** [comment]
@tyveille  - Thank you for the feedback into the issue you are experiencing 

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue in general?
